cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -64,6 +64,7 @@ public final class DataprocUtils {
   public static final String CDAP_CACHED_ARTIFACTS = "cached-artifacts";
   public static final String WORKER_CPU_PREFIX = "Up to";
   public static final String DISABLE_GCS_CACHING = "disableGCSCaching";
+  public static final String ENABLE_GCS_CACHING_SNAPSHOT = "enableGCSCachingSnapshot";
```

Comment:
This gets confused with disableGCSCaching. Can we pick a better name, and also add some comments to line 66 and 67

Reply:
ack